force-app/main/default/layouts/Account-Household Layout.layout-meta.xml

```diff
@@ -118,6 +118,10 @@
                 <behavior>Readonly</behavior>
                 <field>npo02__Best_Gift_Year_Total__c</field>
             </layoutItems>
+            <layoutItems>
```

Comment:
We should chat with Grace about where in the Layouts this new field should be added (if at all).

Reply:
messaged Sam and Grace in WI